client/src/components/MasonryContainer.jsx

```diff
@@ -41,8 +48,14 @@ export default function PaginationTable({ isNewEntry }) {
     setEntries([...sortedDataDesc]);
   };
 
-  const handleDeleteDiary = async (diaryID, editUserId) => {
-    console.log(`${editUserId}`, username);
+  const handleDeleteDiary = async (diaryID, editUserId, image_url) => {
+    console.log(`${editUserId}`, username, `${image_url}`, storage);
+    const imgRef = ref(storage, `${image_url}`);
+    deleteObject(imgRef).then(() => {
```

Comment:
It might be better to name `deleteObject` to specify what it is deleting and from where. I.e `DeleteImageFromFirebaseStorage`

Reply:
Ignore this comment because `deleteObject` is firebase API